lecture-variables.html

```diff
@@ -63,23 +63,36 @@
     <pre><code>var bensFavLanguage = "JavaScript"
 var studentsFavLanguage = "French"
 console.log("Student's favorite language was " + studentsFavLanguage)
-// It was French!
+// Student's favorite language was French
 
 // .... after this lecture ....
 studentsFavLanguage = bensFavLanguage
 console.log("Student's favorite language was " + studentsFavLanguage)
-// Now it's JavaScript!</code></pre>
+// Student's favorite language was JavaScript</code></pre>
+
+    <p>
+        Notice how in the code snippet above our second <code>console.log</code> statement is identical to the first <code>console.log</code> statement but the output is drastically different since the value of the var <code>studentsFavLanguage</code> has changed.
+    </p>
    
     <p>
       As usual, the <a href="http://www.w3schools.com/js/js_variables.asp">w3schools page on variables</a> is a fantastic resource worth reading and visiting often. Here are some other fun facts and features of variables:
       <ul>
         <li>Variables can be used and reused as many times as you want</li>
         <li>Variables can hold different kinds or <b>types</b> of data (Numbers, Strings, and more!)</li>
         <li>The name of a variable is very important for you and others reading your code.</li>
-        <li>The name of a variable can contain letters, numbers, underscores, and dollar signs. However it <i>cannot start with a number</i>.
+        <li>The name of a variable can contain letters, numbers, underscores, and dollar signs. However it <i>cannot start with a number</i>. Also, <b>spaces</b> and <b>dashes</b> are <b>not</b> allowed in a variable name &mdash; having these characters in your variable name will throw an error.
       </ul>
     </p>
 
+    <pre><code>// BAD variable names
+var 3.14isPi; // can't start with number, can't have periods
+var hello everyone!; // space and '!' not allowed
+var this-is-also-not-kosher; // has dashes, too long
```

Comment:
"too long" is an opinion. An important one, but an opinion none-the-less. We should make that clear